sdk/communication/azure-communication-identity/tests/recordings/test_communication_identity_client_async.test_get_token_with_no_scopes.yaml

```diff
@@ -0,0 +1,68 @@
+interactions:
+- request:
+    body: '{}'
+    headers:
+      Accept:
+      - application/json
+      Content-Length:
+      - '2'
+      Content-Type:
+      - application/json
+      Date:
+      - Wed, 21 Apr 2021 01:04:05 GMT
+      User-Agent:
+      - azsdk-python-communication-identity/1.0.0 Python/3.9.0 (Windows-10-10.0.19041-SP0)
+      x-ms-return-client-request-id:
+      - 'true'
+    method: POST
+    uri: https://sanitized.communication.azure.com/identities?api-version=2021-03-07
+  response:
+    body:
+      string: '{"identity": {"id": "sanitized"}}'
+    headers:
+      api-supported-versions: 2020-07-20-preview2, 2021-02-22-preview1, 2021-03-07
+      content-type: application/json; charset=utf-8
+      date: Wed, 21 Apr 2021 01:04:04 GMT
+      ms-cv: +Ld0uuJrsEy+Fmi0FwHE8A.0
+      request-context: appId=
+      strict-transport-security: max-age=2592000
+      transfer-encoding: chunked
+      x-processing-time: 12ms
+    status:
+      code: 201
+      message: Created
+    url: https://lakshmanscommunication-communication.communication.azure.com/identities?api-version=2021-03-07
+- request:
+    body: '{}'
+    headers:
+      Accept:
+      - application/json
+      Content-Length:
+      - '2'
+      Content-Type:
+      - application/json
+      Date:
+      - Wed, 21 Apr 2021 01:04:05 GMT
+      User-Agent:
+      - azsdk-python-communication-identity/1.0.0 Python/3.9.0 (Windows-10-10.0.19041-SP0)
+      x-ms-return-client-request-id:
+      - 'true'
+    method: POST
+    uri: https://sanitized.communication.azure.com/identities/sanitized/:issueAccessToken?api-version=2021-03-07
+  response:
+    body:
+      string: '{"error": {"code": "ValidationError", "message": "Scopes: The Scopes
```

Comment:
Why are we even testing this? `scopes` is a mandatory parameter of this route:
https://github.com/Azure/azure-rest-api-specs/blob/master/specification/communication/data-plane/Microsoft.CommunicationServicesIdentity/stable/2021-03-07/CommunicationIdentity.json#L265

Reply:
This is to make sure that we are indeed getting back a proper error and handling it when passing in an invalid set of input.